SoftwareGuide/Latex/Introduction/Installation.tex

```diff
@@ -589,6 +632,58 @@ \section{Getting Started With ITK}
 to see the output. It is therefore preferable to run the executable from the DOS
 command line by starting the \code{cmd.exe} shell first.
 
+\section{Using ITK as an External Library}
+\label{sec:UsingITKAsExternalLibrary}
+
+For a project that uses ITK as an external library, it is recommended to
+specify the individual ITK modules in the \code{COMPONENTS} argument in the
+\code{find_package} CMake command:
+
+\small
+\begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{bash}
+find_package(ITK COMPONENTS Module1 Module2 REQUIRED)
+include(\${ITK_USE_FILE})
+\end{minted}
+\normalsize
+
+If you would like to use the CMake ExternalProject
+Module\footnote{url{https://cmake.org/cmake/help/latest/module/ExternalProject.html}}
+to download ITK source code when building your ITK application (a.k.a.
+Superbuild ITK), here is a basic CMake snippet for setting up a Superbuild in
+an ITK application project using CMake:
+
+\small
+\begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{bash}
+ExternalProject_Add(ITK
```

Comment:
Somebody with more experience in superbuilds should check this. @thewtex?

Reply:
LGTM